index.html

```diff
@@ -51,9 +51,220 @@ <h2 class="hero-msg">Glad to see you!</h2>
+                <div class="card-job">
+                  <h6>Back End Dev</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>2015</h6>
+                </div>
+              </div>
+              <div class="card-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </div>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="see-link">
+                <a href="#">See Project</a>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="card-container">
+        <div class="card-wrapper">
+          <div class="card">
+            <div class="card-image">
+              <img src="./img/Portfolio4.svg" alt="portfolio" />
+            </div>
+            <div class="card-text-container">
+              <div class="card-text">
+                <h6>Multi-Post Stories</h6>
+              </div>
+              <div class="card-tagline">
+                <div class="card-canopy">
+                  <h6>CANOPY</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>Back End Dev</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>2015</h6>
+                </div>
+              </div>
+              <div class="card-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </div>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="see-link">
+                <a href="#">See Project</a>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <main>
+      <div class="hero">
+        <div>
+          <h1 class="hero-name">About</h1>
+        </div>
+        <div>
+          <h2 class="hero-msg">Myself</h2>
+        </div>
+        <div class="hero-minor-text">
+          <p>
+            I'm a software! I can help you build a product, feature or website
+            Look through some of my work and experience! If you like what you
+            see and have a project you need coded, don't hestiate to contact me.
+          </p>
+        </div>
+      </div>
+      <div>
+        <a href="#" class="text-connect"> LET'S CONNECT </a>
         <div>
```

Comment:
- [ ] Its impressive your assumption for this has been a link, but it isn't. Kindly use a `hx` tag, to follow the best practice on semantically good `html` structure and make our code look professional 👍